ev3dev2/motor.py

```diff
@@ -63,7 +63,10 @@
     from ev3dev2._platform.brickpi import OUTPUT_A, OUTPUT_B, OUTPUT_C, OUTPUT_D
 
 elif platform == 'brickpi3':
-    from ev3dev2._platform.brickpi3 import OUTPUT_A, OUTPUT_B, OUTPUT_C, OUTPUT_D
+    from ev3dev2._platform.brickpi3 import OUTPUT_A, OUTPUT_B, OUTPUT_C, OUTPUT_D, 
+                                           OUTPUT_E, OUTPUT_F, OUTPUT_G, OUTPUT_H,
```

Comment:
I think you need to escape the newline

Reply:
I don't in my python code I'll do it here.